src/v2/guide/security.md

```diff
@@ -21,65 +21,71 @@ new Vue({
 })
 ```
 
-Vue templates are compiled into JavaScript, and expressions inside templates will be executed as part of the rendering process. Although the expressions are evaluated against a specific rendering context, due to the complexity of potential global execution environments, it is impractical for a framework like Vue to completely shield you from potential malicious code execution without incurring unrealistic performance overhead. The most straightforward way to avoid this category of problems altogether is to make sure the contents of your Vue templates are always trusted and entirely controlled by you.
+Vue templates are compiled into JavaScript, and expressions inside templates are executed as part of the rendering process. Although the expressions are evaluated against a specific rendering context, due to the complexity of potential global execution environments, it is impractical for a framework like Vue to completely shield you from potential malicious code execution without incurring unrealistic performance overhead. The most straightforward way to avoid this category of problems altogether is to make sure the contents of your Vue templates are always trusted and entirely controlled by you.
 
 ## What Vue Does to Protect You
 
 ### HTML content
 
-Whether using templates or render functions, content is automatically escaped. That means in this template:
+Whether using templates or render functions, content is automatically escaped.
+
+Consider the following template:
 
 ```html
 <h1>{{ userProvidedString }}</h1>
 ```
 
-if `userProvidedString` contained:
+If the value of `userProvidedString` was this:
 
 ```js
 '<script>alert("hi")</script>'
 ```
 
-then it would be escaped to the following HTML:
+Then it would be escaped to the following HTML:
 
 ```html
 &lt;script&gt;alert(&quot;hi&quot;)&lt;/script&gt;
 ```
 
-thus preventing the script injection. This escaping is done using native browser APIs, like `textContent`, so a vulnerability can only exist if the browser itself is vulnerable.
+Thus preventing the script injection.
+
+This escaping is done using native browser APIs (like `textContent`), so a vulnerability can only exist if the browser itself is vulnerable.
```

Comment:
This whole rewritten section, including the change of cases, would break the continuity of the text.